src/Compilers/Core/Portable/Emit/Context.cs

```diff
@@ -19,5 +20,27 @@ public EmitContext(CommonPEModuleBuilder module, SyntaxNode syntaxNodeOpt, Diagn
             SyntaxNodeOpt = syntaxNodeOpt;
             Diagnostics = diagnostics;
         }
+
+        public bool Filter(IMethodDefinition method)
```

Comment:
>public bool Filter(IMethodDefinition method) [](start = 8, length = 44)

I think we should keep this type data-only and place the helpers in the layer that needs them.